src/components/atoms/DigitInput/DigitInput.stories.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable arrow-body-style */
+import React, { useState } from 'react';
+import { Meta } from '@storybook/react';
+import DigitInput from './DigitInput';
+
+export default {
+  component: DigitInput,
+  title: 'atoms/DigitInput',
+} as Meta;
+
+export const Default = () => {
+  const [value, setValue] = useState('');
+  const handleChange = (event: { target: { value: string; }; }) => {
```

Comment:
```suggestion
  const handleChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
```
이 부분은 event가 implicitly any라서 이렇게 쓰신 걸까요? 그러면 이렇게 고치는 편이 더 깔끔할 것 같습니다!

Reply:
제가 Quick Fix를 하면서 임시적으로 두었던 type값이 그대로 있는 실수를 했습니다. 수정하겠습니다.